tests/io/vasp/test_sets.py

```diff
@@ -750,6 +751,116 @@ def test_grid_size_from_struct(self):
             assert matched
 
 
+class TestMatPESStaticSet(PymatgenTest):
+    def setUp(self):
+        self.tmp = tempfile.mkdtemp()
+
+    def test_init(self):
+        prev_run = f"{TEST_FILES_DIR}/relaxation"
+
+        vis = MatPESStaticSet.from_prev_calc(prev_calc_dir=prev_run)
+        # Check that INCAR settings were load from yaml, check the default functional is PBE.
+        assert vis.incar["NSW"] == 0
+        assert vis.incar["NELM"] == 200
+        assert vis.incar["ENCUT"] == 680
+        assert vis.incar["ENAUG"] == 1360
+        assert vis.incar["EDIFF"] == 1.0e-05
+        assert vis.incar["LREAL"] is False
+        assert vis.incar["LORBIT"] == 11
+        assert vis.incar["LVHAR"]
+        assert vis.incar["ISMEAR"] == 0
+        assert vis.incar["KSPACING"] == 0.22
+        assert vis.incar["GGA"] == "PE"
```

Comment:
No need to assert keys one by one. If you want to make sure they match the values in `pymatgen/io/vasp/MatPESStaticSet.yaml`, you can load that file into a dict and compare `vis.incar` against that dict.